vaadin-spring/src/test/java/com/vaadin/flow/spring/SpringSecurityAutoConfigurationTest.java

```diff
@@ -151,11 +182,27 @@ private static void assertThatCustomNavigationAccessCheckerIsUsed(
         Mockito.verify(navigationContext, Mockito.never()).neutral();
     }
 
+    private <T> void assertObjectIsSerializable(T instance) {
+        Object deserialized = Assertions.assertDoesNotThrow(() -> {
+            ByteArrayOutputStream bs = new ByteArrayOutputStream();
+            ObjectOutputStream out = new ObjectOutputStream(bs);
```

Comment:
Intellij like this version better:
```suggestion
            try (ObjectOutputStream out = new ObjectOutputStream(bs)) {
                out.writeObject(instance);
            }
```

Maybe we could use it.

Reply:
done

Reply:
Thanks, Marco!